sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/ITokenStorage.java

```diff
@@ -0,0 +1,26 @@
+package com.microsoft.appcenter.utils.context;
+
+/**
+ * Interface for storage that works with token.
+ */
+public interface ITokenStorage {
+
+    /**
+     * Stores token value.
+     *
+     * @param token auth token.
+     */
+    void saveToken(String token);
+
+    /**
+     * Retrieves token value.
+     *
+     * @return auth token.
+     */
+    String getToken();
+
+    /**
+     * Removes token value.
+     */
+    void removeToken();
```

Comment:
Why do we need a `removeToken` method? I don't see it used anywhere

Reply:
Update: I think we need it, but please call it when we disable the Identity service